.gitpod.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full
+
+USER root
```

Comment:
From https://github.com/techwithtim/Account-Storage/commit/e09d540cae505ef3c7a7be1f6544a85471baa338#diff-0d7421b92d33be6626e36fa9c2e9dbd1R3, 

https://community.gitpod.io/t/rails-app-trying-to-get-chromedriver-working/1268

Reply:
@nisarhassan12 I'm sorry to bother you but I'm getting this `Operation not permitted` error and wondered if you knew anything about this? I had to try adding dependencies to match the `.travis.yml` file because it was missing `libxi-dev`. I found some references to work from but it doesn't seem to be working.

```
building base image

initializing base image context
UGit operations complete                       location=/workspace/init
running base image build
Step 1/4 : FROM gitpod/workspace-full
latest: Pulling from gitpod/workspace-full
Digest: sha256:a286eacd645a06c022f03a1edde2d8976877a146fd56e14ecf543418d0a6f3ab
Status: Image is up to date for gitpod/workspace-full:latest
 ---> 8bdab2c1e6e8
Step 2/4 : USER root
 ---> Using cache
 ---> 05cc90489a0d
Step 3/4 : RUN sudo apt-get update  && sudo apt-get install -y     install xserver-xorg-dev libxext-dev libxi-dev build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev  && sudo rm -rf /var/lib/apt/lists/*
 ---> Running in 75fb3e2dfbb5
sudo: setrlimit(RLIMIT_CORE): Operation not permitted
Get:1 https://apt.llvm.org/focal llvm-toolchain-focal InRelease [4,231 B]
Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [107 kB]
Get:3 http://ppa.launchpad.net/git-core/ppa/ubuntu focal InRelease [23.8 kB]
Get:4 https://apt.llvm.org/focal llvm-toolchain-focal/main amd64 Packages [10.6 kB]
Hit:5 http://archive.ubuntu.com/ubuntu focal InRelease
Get:6 http://archive.ubuntu.com/ubuntu focal-updates InRelease [107 kB]
Get:7 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [33.4 kB]
Get:8 http://ppa.launchpad.net/git-core/ppa/ubuntu focal/main amd64 Packages [4,693 B]
Get:9 http://archive.ubuntu.com/ubuntu focal-backports InRelease [98.3 kB]
Get:10 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [44.5 kB]
Get:11 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [160 kB]
Get:12 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [1,077 B]
Get:13 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [303 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [155 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [1,077 B]
Get:16 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [33.9 kB]
Get:17 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [2,900 B]
Fetched 1,091 kB in 2s (612 kB/s)
Reading package lists...
sudo: setrlimit(RLIMIT_CORE): Operation not permitted
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package install



build failed: cannot build base image: The command '/bin/sh -c sudo apt-get update  && sudo apt-get install -y     install xserver-xorg-dev libxext-dev libxi-dev build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev  && sudo rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100

Error: build failed: cannot build base image: The command '/bin/sh -c sudo apt-get update  && sudo apt-get install -y     install xserver-xorg-dev libxext-dev libxi-dev build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev  && sudo rm -rf /var/lib/apt/lists/*' retu
```

Reply:
Ah! https://stackoverflow.com/a/38004106:

> Because you have an https sources. Install `apt-transport-https` before executing update.

Reply:
OK, now we seem past that Permissions error and are on to: 

```
Reading state information...
E: Unable to locate package install

build failed: cannot build base image: The command '/bin/sh -c sudo apt-get update && apt-get install -y apt-transport-https  && sudo apt-get install -y     install xserver-xorg-dev libxext-dev libxi-dev build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev  && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/*' returned a non-zero code: 100
```

This is odd: see https://askubuntu.com/a/1164538 

> The reason you are seeing this error is because you are trying to install a package called "install" (next to cmake and libelf-dev). This looks like a simple typo. Just remove it and it should be fixed, because "install" is not a package.

Reply:
Oh haha oops: `sudo apt-get install -y     install`